### PR TITLE
Refactor migrations as a single migration

### LINKED ISSUE #277: Replace all migrations with single migration
### Pre-requisites:
- [x] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find another request for the same or similar feature.

## Is your feature request related to a problem, or an open issue? Describe the problem, and if applicable, reference the issue.

We need to clean up our migration history

## Describe the solution you'd like

Delete all migrations, drop your database, re-create your database, and then run the command in the README to generate all the tables in a single migration.

### PR DESCRIPTION
**Pull request by mkn170030**
Added a comment on /src/components/RegisteredTeamsList.tsx line 45, added a file explaining the planned process for the overhaul of /src/migration as well as adding my name to Authors.md

## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

## Description of Changes
Added a comment to line 45 of /src/components/RegisteredTeamsList.tsx noting that a bug is caused there.
Added a file to /src/migration detailing the steps to remake the migration files.
Added my name to Authors.md.

## Related Issues
Closes #277 - Replace all migrations with single migration

Co-authored-by: Razvan Preotu \<50164789+RazvanPreotu99@users.noreply.github.com>

### CODE REVIEW

File: src/migration/issue_227_draft.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Steps necessary:
+    Delete all files in this folder.
+    Drop database(? Not 100% sure what this entails. Does this require me to empty the postgres database or to simply delete it?)
```
Comment (SpencerKaiser) [inline src/migration/issue_227_draft.txt:3]: Use the PG cli, PG Admin, or the Postgres App for Mac and literally delete the `hangar` database and then recreate it

File: src/migration/issue_227_draft.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    Re-create database (Presumably empty)
```
Comment (SpencerKaiser) [inline src/migration/issue_227_draft.txt:4]: Yep, make a new empty DB

File: src/migration/issue_227_draft.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    Run the following command:
+        'npm run typeorm migration:run'
```
Comment (SpencerKaiser) [inline src/migration/issue_227_draft.txt:6]:
```suggestion
```

File: src/migration/issue_227_draft.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+        'npm run typeorm migration:generate -- -n AllMigrationsInOne'
```
Comment (SpencerKaiser) [inline src/migration/issue_227_draft.txt:7]:
```suggestion
        'npm run typeorm migration:generate -- -n CreateAllTables'
```

File: src/migration/issue_227_draft.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+    From my understanding, there is no need to change the actual .ts files in the /entities folder, as the databse itself is not supposed to change.
```
Comment (SpencerKaiser) [inline src/migration/issue_227_draft.txt:8]: Correct!

**Review by SpencerKaiser — COMMENTED** [review]
@johnkahn did I miss anything?

**Comment by mkn170030** [comment]
I made a bunch of sloppy mistakes on the git end (I screwed up the branching and probably should have made separate branches for the two issues), but I think #277 is done now? I'm under the impression that the draft PR automatically includes the changes made to this branch as a PR.

File: Authors.md [hunk 6]
```diff
@@ -9,4 +9,5 @@
 - Austin Slobodnik (austinslobodnik@gmail.com)
 - Osman Findik (osmanfindik1@windowslive.com)
 - Maha Siddique (mxs151731@utdallas.edu)
-- Dhruv Tailor (dhruv6198@gmail.com)
\ No newline at end of file
+- Dhruv Tailor (dhruv6198@gmail.com)
+- Michael Nguyen (mkn170030@utdallas.edu)
```
Comment (SpencerKaiser) [inline Authors.md:13]:
```suggestion
- Michael Nguyen (mkn170030@utdallas.edu)
- Razvan Preotu (rnp170130@utdallas.edu)
```

**Review by SpencerKaiser — APPROVED** [review]
:shipit:

**Review by SpencerKaiser — APPROVED** [review]
lgtm

**Review by SpencerKaiser — APPROVED** [review]
lgtm

**Pull request merged**